modules/spring-config/src/main/java/org/mule/config/spring/parsers/specific/RetryNotifierDefinitionParser.java

```diff
@@ -19,5 +23,29 @@ public RetryNotifierDefinitionParser(Class clazz)
     {
         super("notifier", clazz);
     }
+    
+    @Override
+    protected boolean isChild(Element element, ParserContext parserContext, BeanDefinitionBuilder builder)
+    {
+        if (isConfigElement(element))
+        {
+            return false;
+        }
+        else
+        {
+            return true;
+        }
+    }
+
+    private boolean isConfigElement(Element element)
+    {
+        return getAncestorBeanName(element).equals(OBJECT_MULE_CONFIGURATION);
+    }
+
+    private String getAncestorBeanName(Element element)
+    {
+        return ((Element) ((Element) element.getParentNode()).getParentNode()).getAttribute(ATTRIBUTE_NAME);
```

Comment:
No. In case it is in the global configuration, this should have this form:

<configuration>
<reconnect frequency="500">
<reconnect-notifier/>
</reconnect>
</configuration>

A same strategy is used to determine if it is a global configuration for the scripting parser.